.make/Makefile.lnx

```diff
@@ -21,6 +21,5 @@ GOCYCLO_BIN=$(GOCYCLO_DIR)/gocyclo
 GIT_BIN_NAME := git
 GLIDE_BIN_NAME := glide
 GO_BIN_NAME := go
-HG_BIN_NAME := hg
```

Comment:
Why remove HG? Go in general can use any mercurial based repository, and anything we depend on could technically  be, or their dependencies..  (unless this has changed in go?)...

Removing it now seems like just opening up for adding it later... We added it because we previously had a HG dependency in there somewhere.